packages/garbo/src/tasks/dailyFamiliars.ts

```diff
@@ -60,18 +59,15 @@ function drivebyValue(): number {
 }
 
 function entendreValue(): number {
-  const targets = copyTargetCount();
+  const targets = highMeatMonsterCount("Scepter"); // Scepter can cause circular logic
```

Comment:
could you do a quick refactor to both have `highmeatMonsterCount` passed in rather than calculated both times, and to deduplicate some of the logic here?

Reply:
`highMeatMonsterCount` is an expensive calculation to make; this is a small micro-optimization, but instead of calculating it in both `entendreValue` and `drivebyValue`, we can calculate it once (during the robort it task) and pass that value into both functions, saving us some computational power. It's not really saving us much, but it's one of those things where making a habit of calling that function sparingly will add up if we're consistent.

Also, I know why scepter causes circular issues with like diet, but I have no recollection why it gets funky with robort code--might be worth updating the comment to mention why, if you remember